scripts/stress/claim-master.js

```diff
@@ -18,30 +18,20 @@ const KyberStakingWithgetStakerDataForPastEpoch = artifacts.require(
 const {expect} = require('chai');
 const {
   expectEvent,
-  expectRevert,
   balance,
   ether,
+  BN
 } = require('@openzeppelin/test-helpers');
 
```

Comment:
Not related to this line, but when trying to run the stress tests it fails with ` Could not find artifacts for KyberPoolMasterWithClaimedPoolRewardSetter from any sources`.

Also, it looks like `KyberDAOWithRewardPercentageSetter` and `KyberFeeHandlerWithRewardPerEposhSetter` are not used in the tests

Reply:
That is being addressed in PR #22